github/compliance/issue.py

```diff
@@ -234,20 +237,38 @@ def create_or_update_issue(
     latest_processing_date: datetime.date|datetime.datetime=None,
     extra_labels: typing.Iterable[str]=None,
     preserve_labels_regexes: typing.Iterable[str]=(),
+    ctx_labels: typing.Iterable[str]=(),
 ) -> github3.issues.issue.ShortIssue:
+    '''
+    Creates or updates a github issue for the given scanned_element. If no issue exists, yet, it will
+    be created, otherwise, it is checked that at most one matching issue exists; which will then be
+    updated.
+    Issues are found by labels. Some of those are derived from the scanned_element. If given,
+    ctx_labels are also included in search-query (i.e. an issue must have all of the given ctx_labels
+    to be considered a match). All of those labels + any given extra_labels are assigned to the
+    issue, both in case it is created anew, or updated. Extra labels that are not ignored via the
+    preserve_labels_regexes argument are dropped.
```

Comment:
need to add all labels starting w/ ctx-prefix to "labels to preserve", do we not?